docs/source/config-file.rst

```diff
@@ -77,7 +77,7 @@ The config file is made up of multiple parts
     * :code:`'component'`
         * Downstream component to sync with
     * :code:`sync`
-        * This array contains information on what to sync from upstream repos (i.e. 'issue' and/or 'pullreuest')
+        * This array contains information on what to sync from upstream repos (i.e. 'issue' and/or 'pullrequest')
```

Comment:
My editor is configured to automatically remove trailing white spaces. Let me know if this makes this change to noisy and I can isolate this change.